src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -33,4 +39,106 @@ public void getTypeEqualsHouse() {
         assertThat(structure.getType()).isEqualTo(StructureType.HOUSE);
     }
 
+    @Test
+    public void deliverItemTest() {
+        // Arrange
+        final House house = new House(new Position());
+        final IItem item = new IItem() {
+        };
+
+        // Act
+
+        // Assert
+        assertThat(house.deliverItem(item)).isTrue();
```

Comment:
The acting is the `house.deliverItem(item)`. The asserting is the `assertThat(itemDelivered).isTrue();`